### PR TITLE
TK-3: Configure SCSS

### PR DESCRIPTION
**Pull request by DavidB208**
SASS was installed in the project.
Just was modified package.json.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
+    "sass": "^1.54.0",
```
Comment (bruno9926) [inline package.json:12]: dfafdafdasfdas

**Comment by bruno9926** [comment]
Otro comentario

File: package.json (hunk 1 above)
Comment (DavidB208) [reply package.json:12]: jeje

**Comment by bruno9926** [comment]
Probaste con un componente? :B

File: package.json (hunk 1 above)
Comment (bruno9926) [inline package.json:12]: probar con un componente

**Review by bruno9926 — CHANGES_REQUESTED** [review]

**Review by bruno9926 — APPROVED** [review]

**Pull request merged**